comp/remote-config/rcclient/rcclient.go

```diff
@@ -104,25 +104,70 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig)
 		return
 	}
 
-	// TODO RCM-1064: implement priority between CLI and remote-config
-	// If there is no error, override the configs
-	if len(mergedConfig.LogLevel) > 0 && mergedConfig.LogLevel != rc.configState.LatestLogLevel {
-		pkglog.Infof("Changing log level to %s through remote config", mergedConfig.LogLevel)
+	// Checks who (the source: rc, cli, or default) is responsible for the last logLevel change
+	// If the logLevel has been set by "cli", we do NOT change anything
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I think you can rephrase that by saying the priority is `CLI > RC > Default`